src/uiTest/scala/com/virtuslab/gitmachete/uitest/RunningIntelliJPerSuite.scala

```diff
@@ -6,25 +6,7 @@ import org.virtuslab.ideprobe.RunningIntelliJPerSuiteBase
 trait RunningIntelliJPerSuite extends RunningIntelliJPerSuiteBase {
```

Comment:
Actually... check `org.virtuslab.ideprobe.junit4.RunningIntelliJPerSuite`, I think this trait can be completely removed from our project now (it was only added as a copy of `org.virtuslab.ideprobe.junit4.RunningIntelliJPerSuite` but with ignore-errors feature)